projects/wave-core-new/src/lib/operators/datatype.model.ts

```diff
@@ -22,6 +24,8 @@ export abstract class DataType {
      * @return the smallest value.
      */
     abstract getMax(): number;
+
+    abstract no_data(value: number): NoDataDict;
```

Comment:
I also struggle with this, but I guess we should use camel case in our JS world :grinning: .

Reply:
0365c2d4a4bb46141542e9a67e1c4f961c2e097a